<?xml version="1.0"?>
<launch>
    <param name="use_sim_time" value="false" />

    <arg name="dwa_param" default="$(find pid_dwa_control)/config/dwa_param.yaml"/>
    <arg name="robot_param" default="$(find pid_dwa_control)/config/robot_param.yaml"/>

    <node pkg="pid_dwa_control" type="pid_dwa_control_node" name="pid_dwa_control_node" output="screen">
        <rosparam command="load" file="$(arg dwa_param)"/>
        <rosparam command="load" file="$(arg robot_param)"/>

        <param name="enable_back" value="false"/>
        <param name="max_linear_velocity" value="0.3"/>
        <param name="max_angular_velocity" value="0.5"/>
        <param name="gain_linear_velocity" value="0.6"/>
        <param name="gain_angular_velocity" value="2.5"/>
        <param name="distance" value="1.5"/>
        <param name="timeout" value="1.0"/>
        <param name="scan_angle_resolution" value="0.087"/>
    </node>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find pid_dwa_control)/config/mono_tracking_ModelOnly.rviz" output="screen"/>

</launch>